post-processing/streamlit_post_processing.py

```diff
@@ -169,6 +177,23 @@ def axis_select(label: str, axis: dict):
 
     # units select
     units_select(label, axis)
+
+    # FIXME: add ability to use a custom value for only one of min or max
+    disable_user_range = plot_type != 'line'
```

Comment:
```suggestion
    disable_user_range = plot_type != "line" or axis.get("value") is None
```
I noticed an error today where the web app isn't happy if you try to set a custom range before selecting which axis column you'd like to use. I reckon the easiest way to deal with this is to just not allow the scenario to happen and keep custom ranges disabled until an axis column is selected.